test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -3,40 +3,37 @@
 # Copyright (C) 1994-2020 Altair Engineering, Inc.
 # For more information, contact Altair at www.altair.com.
 #
-# This file is part of both the OpenPBS software ("OpenPBS")
-# and the PBS Professional ("PBS Pro") software.
+# This file is part of the PBS Professional ("PBS Pro") software.
```

Comment:
@jdburton Looks like merge/rebase issue in this file, please revert changes in this file.

Reply:
Yes, it was a merge issue. I restored the file to what is in master because no changes are necessary.